textile/features.textile

```diff
@@ -2349,6 +2361,9 @@ class ReferenceExtras: // REX*
   timeserial: String // REX2a
   type: String //REX2b
 
+class ClientLibraryInfo: // CLI*
+  +agents: Dict<String, String?> // CLI2
+  +agentIdentifier(additionalAgents: Dict<String, String?>?) => String // CLI3; support for the additionalAgents parameter is optional
```

Comment:
How is it supposed to provide these `additionalAgents` to `ClientOptions`, which now only accepts dictionary as an input?

Reply:
I'm not sure what you mean here. There’s no interaction between `ClientLibraryInfo` and `ClientOptions`… would you be able to explain in a bit more detail please?

Reply:
resolved in DM